### PR TITLE
update for gitlab

### PR DESCRIPTION
**Pull request by joshmarinacci**

### CODE REVIEW

File: flaky-tests/ci-providers/gitlab.md [hunk 1]
```diff
@@ -6,10 +6,10 @@ description: Configure Flaky Tests using GitLab CI
 
 ## Getting Started
 
-You can use the analytics test uploader within your [GitLab Pipelines](https://docs.gitlab.com/ee/ci/pipelines/) to upload your test results.
+You can use the analytics test cli within your [GitLab Pipelines](https://docs.gitlab.com/ee/ci/pipelines/) to upload your test results.
```
Comment (dfrankland) [inline flaky-tests/ci-providers/gitlab.md:9]: nit: Can we change all lowercase `cli` to uppercase `CLI` when it's used in plain-English? 🙏

```suggestion
You can use the analytics test CLI within your [GitLab Pipelines](https://docs.gitlab.com/ee/ci/pipelines/) to upload your test results.
```

File: flaky-tests/ci-providers/gitlab.md [hunk 2]
```diff
@@ -26,18 +26,21 @@ Next you will need your Trunk **organization slug** and **token.** Navigate to [
 
 [Create a GitLab Variable](https://docs.gitlab.com/ee/ci/variables/index.html#for-a-project) for your `TRUNK_API_TOKEN` by navigating to your GitLab project's **Settings** > **CI/CD** > **Variables**. You will use this variable in the next steps to upload test results to Flaky Tests.
 
-### Add Uploader to Testing Workflow
+### Add Analytics CLI to Testing Workflow
 
-You can upload test results to Flaky Tests with the [`trunk-analytics-uploader`](https://github.com/trunk-io/analytics-uploader) by running it in a stage after your tests are complete. There are five different OS/arch builds of the uploader in the latest release. Pick the one you need for your testing platform and be sure to download the release on every CI run. **Do not bake the CLI into a container or VM.** This ensures your CI runs are always using the latest build.
+You can upload test results to Flaky Tests with the [`trunk-analytics-cli`](https://github.com/trunk-io/analytics-cli) by running
+it in a stage after your tests are complete. There are four different OS/arch builds of the cli in the latest release. Pick the
```
Comment (dfrankland) [inline flaky-tests/ci-providers/gitlab.md:32]: nit: another instance of `cli`

```suggestion
it in a stage after your tests are complete. There are four different OS/arch builds of the CLI in the latest release. Pick the 
```

File: flaky-tests/ci-providers/gitlab.md [hunk 3]
```diff
@@ -6,10 +6,10 @@ description: Configure Flaky Tests using GitLab CI
 
 ## Getting Started
 
-You can use the analytics test uploader within your [GitLab Pipelines](https://docs.gitlab.com/ee/ci/pipelines/) to upload your test results.
+You can use the analytics test cli within your [GitLab Pipelines](https://docs.gitlab.com/ee/ci/pipelines/) to upload your test results.
 
 {% hint style="info" %}
-The Trunk Flaky Tests Uploader currently only supports Linux x64 and macOS for Intel and Arm. If you have another use case, please get in touch with support at [https://slack.trunk.io](https://slack.trunk.io/). For the best results, you'll need to validate that your test invocation doesn't use cached test results and doesn't automatically retry failing tests.
+The Trunk Flaky Tests CLI currently only supports Linux x64 and macOS for Intel and Arm. If you have another use case, please get in touch with support at [https://slack.trunk.io](https://slack.trunk.io/). For the best results, you'll need to validate that your test invocation doesn't use cached test results and doesn't automatically retry failing tests.
```
Comment (dfrankland) [inline flaky-tests/ci-providers/gitlab.md:12]: We support Arm for linux now 👍 

```suggestion
The Trunk Flaky Tests CLI currently only supports Linux for x64 and Arm, and macOS for Intel and Arm. If you have another use case, please get in touch with support at [https://slack.trunk.io](https://slack.trunk.io/). For the best results, you'll need to validate that your test invocation doesn't use cached test results and doesn't automatically retry failing tests.
```

File: flaky-tests/ci-providers/gitlab.md [hunk 4]
```diff
@@ -56,14 +59,14 @@ unit-test-job:   # This job runs the tests
 upload_test_results: # This job uploads tests results run in the last stage
   stage: flaky-tests
   script:
-    - curl -fsSL --retry 3 "UPLOADER_LINK" -o ./trunk-analytics-uploader
-    - chmod +x ./trunk-analytics-uploader
-    - ./trunk-analytics-uploader upload --junit-paths "tests/jest/jest_junit_test.xml" --org-url-slug <TRUNK_ORG_SLUG> --token $TRUNK_API_TOKEN
+    - curl -fsSL --retry 3 "CLI_LINK" | tar -xvf > ./trunk-analytics-cli
```
Comment (dfrankland) [inline flaky-tests/ci-providers/gitlab.md:62]: I copy-pasted the wrong command to you 🤦 my apologies
```suggestion
    - curl -fsSL --retry 3 "CLI_LINK" | tar -xvz > ./trunk-analytics-cli
```

File: flaky-tests/ci-providers/gitlab.md [hunk 5]
```diff
@@ -56,14 +59,14 @@ unit-test-job:   # This job runs the tests
...
+    - chmod +x ./trunk-analytics-cli
+    - ./trunk-analytics-cli upload --junit-paths "tests/jest/jest_junit_test.xml" --org-url-slug <TRUNK_ORG_SLUG> --token $TRUNK_API_TOKEN
```
Comment (dfrankland) [inline flaky-tests/ci-providers/gitlab.md:64]: an aside: Anyone can test this out locally by trying to run `./trunk-analytics-cli --version` 😄

File: flaky-tests/ci-providers/buildkite-quickstart.md [hunk 6]
```diff
@@ -24,16 +24,19 @@ Next you will need your Trunk **organization slug** and **token.** Navigate to [
...
 | macOS Intel         | [x86\_64-apple-darwin](https://github.com/trunk-io/analytics-cli/releases/latest/download/trunk-analytics-cli-x86\_64-apple-darwin.tar.gz)        |
 | macOS Apple Silicon | [aarch64-apple-darwin](https://github.com/trunk-io/analytics-cli/releases/latest/download/trunk-analytics-cli-aarch64-apple-darwin.tar.gz)        |
-| Arm64 Linux         | [aarch64-unknown-linux-musl](https://github.com/trunk-io/analytics-cli/releases/latest/download/trunk-analytics-cli-aarch64-unknown-linux.tar.gz) |
-| Intel Linux         | [x86\_64-unknown-linux-musl](https://github.com/trunk-io/analytics-cli/releases/latest/download/trunk-analytics-cli-x86\_64-unknown-linux.tar.gz) |
+| Arm64 Linux         | [aarch64-unknown-linux](https://github.com/trunk-io/analytics-cli/releases/latest/download/trunk-analytics-cli-aarch64-unknown-linux.tar.gz) |
+| Intel Linux         | [x86\_64-unknown-linux](https://github.com/trunk-io/analytics-cli/releases/latest/download/trunk-analytics-cli-x86\_64-unknown-linux.tar.gz) |
```
Comment (mmatheson) [inline flaky-tests/ci-providers/buildkite-quickstart.md:39]: This link 404s

File: flaky-tests/ci-providers/buildkite-quickstart.md [hunk 7]
```diff
@@ -44,16 +47,16 @@ steps:
     key: tests
   - label: "Upload test results"
     commands:
-      - curl -fsSL --retry 3 "UPLOADER_LINK" -o ./trunk-analytics-uploader
-      - chmod +x ./trunk-analytics-uploader
-      - ./trunk-analytics-uploader upload --junit-paths *.xml --org-url-slug trunk --token $$TRUNK_TOKEN
+    - curl -fsSL --retry 3 "CLI_LINK" | tar -xvz > ./trunk-analytics-cli
+      - chmod +x ./trunk-analytics-cli
```
Comment (mmatheson) [inline flaky-tests/ci-providers/buildkite-quickstart.md:51]: Did you test this? The resulting binary should already be executable

File: flaky-tests/ci-providers/buildkite-quickstart.md [hunk 8]
```diff
@@ -44,16 +47,16 @@ steps:
...
+      - ./trunk-analytics-cli upload --junit-paths *.xml --org-url-slug trunk --token $$TRUNK_TOKEN
```
Comment (mmatheson) [inline flaky-tests/ci-providers/buildkite-quickstart.md:52]: `--trunk-token` is not an option. You can double check this by following these docs and running `./trunk-analytics-cli upload --help`:

`Usage: trunk-analytics-cli upload [OPTIONS] --junit-paths <JUNIT_PATHS> --org-url-slug <ORG_URL_SLUG> --token <TOKEN>`

**Review by mmatheson — CHANGES_REQUESTED** [review]

File: flaky-tests/ci-providers/buildkite-quickstart.md (hunk 7 above)
Comment (joshmarinacci) [reply flaky-tests/ci-providers/buildkite-quickstart.md:51]: I did test that the download worked. I didn't test if it was already executable. Does tar.gz preserve the executable bit on mac and linux?

File: flaky-tests/ci-providers/buildkite-quickstart.md (hunk 6 above)
Comment (joshmarinacci) [reply flaky-tests/ci-providers/buildkite-quickstart.md:39]: which one doesn't work? They both worked for me.   Is it the second one that has the `\_` escape in it? That is removed when the markdown is rendered.

File: flaky-tests/ci-providers/buildkite-quickstart.md (hunk 7 above)
Comment (joshmarinacci) [reply flaky-tests/ci-providers/buildkite-quickstart.md]: I removed the chmod lines.

File: flaky-tests/ci-providers/buildkite-quickstart.md (hunk 8 above)
Comment (joshmarinacci) [reply flaky-tests/ci-providers/buildkite-quickstart.md]: I removed that extra `trunk` option.

**Review by mmatheson — APPROVED** [review]

**Pull request merged**